optimizely/event/event_processor.py

```diff
@@ -180,14 +180,15 @@ def _run(self):
     """
         try:
             while True:
-                if self._get_time() >= self.flushing_interval_deadline:
+                loop_time = self._get_time()
+                if loop_time >= self.flushing_interval_deadline:
                     self._flush_batch()
                     self.flushing_interval_deadline = self._get_time() + \
```

Comment:
Should we change this ._get_time() to loop_time as well? - line 186

Reply:
Was thinking about that yes. Thx for the pointer. Will implement and test.